### ISSUE TITLE
wrong static file path in history mode

### ISSUE THREAD

**Issue by liuhahi** [creativetimofficial/vue-paper-dashboard-pro#26]
![screen shot 2017-11-14 at 11 23 25 am](https://user-images.githubusercontent.com/2950675/32761441-4bccee2c-c92e-11e7-871a-860254b65bf2.png)

The static files path are configured wrongly in history mode.

Solution:  
Change from      
logo: {
        type: String,
        default: 'static/img/vue-logo.png'
      },

to 

logo: {
        type: String,
        default: '/static/img/vue-logo.png'
      },

Basically, need to add '/' to all the static files.

**Comment by liuhahi** [comment]
@sfdye

**Comment by cristijora** [comment]
Hi @liuhahi 

As much as I would like to solve this problem, unfortunately I can't. 
The initial provided setup in this template makes it possible to build the theme as a static website which means you can deploy it pretty much anywhere.

If you use vue-router in history mode, it means that the theme becomes a node.js powered website and thus can be deployed as a node app only. 

This has been discussed in another issue an unfortunately it will remain like it is. There is a new section in the [documentation](https://cristijora.github.io/paper-dashboard-pro/documentation.html?selectedKind=General%20tips&selectedStory=Deployment%20tips&full=0&down=1&left=1&panelRight=0&downPanel=storybook%2Factions%2Factions-panel)

If you need vue-router in history mode simply add a `/` before each `static` referenced asset and change 
`assetsPublicPath: ''` to `assetsPublicPath: '/'` inside `config/index.js`

**Comment by liuhahi** [comment]
@cristijora Okay Noted with thanks! :)
